icaruscode/Timing/PMTTimingCorrectionsProvider.cxx

```diff
@@ -207,16 +208,17 @@ void icarusDB::PMTTimingCorrectionsProvider::readTimeCorrectionDatabase(const ar
     if( fVerbose ) {
 
         mf::LogInfo(fLogCategory) << "Dump information from database " << std::endl;
-        mf::LogInfo(fLogCategory) << "channel, trigger cable delay, reset cable delay, laser corrections, muons corrections" << std::endl;
-
+	std::ostringstream ss;
+        ss << "channel, trigger cable delay, reset cable delay, laser corrections, muons corrections" << std::endl;
         for( auto const & [key, value] : fDatabaseTimingCorrections ){
-            mf::LogInfo(fLogCategory) << key << " " 
-                  << value.triggerCableDelay << "," 
-                  << value.resetCableDelay << ", " 
-                  << value.laserCableDelay << ", "
-                  << value.cosmicsCorrections << ","
-                  << std::endl; 
+            ss << key << " " 
+               << value.triggerCableDelay << "," 
+               << value.resetCableDelay << ", " 
+               << value.laserCableDelay << ", "
+               << value.cosmicsCorrections << ","
+               << std::endl; 
         }
+	mf::LogInfo(fLogCategory) << ss.str();
     }
```

Comment:
I assume this change is to avoid the header of messagefacility on each single line. Two ways to proceed here.
You may want to create a "permanent" instance of the `mf::LogInfo` object:
```cpp
      mf::LogInfo log(fLogCategory);
      log << "Dump information from database "
        << "\nchannel, trigger cable delay, reset cable delay, laser corrections, muons corrections";
      for( auto const & [key, value] : fDatabaseTimingCorrections ){
         log << "\n" << key << " "
            << ", " << value.triggerCableDelay
            << ", " << value.resetCableDelay
            << ", " << value.laserCableDelay
            << ", " << value.cosmicsCorrections;
      }
```
This will enclose the whole message in a single header.
Or, you may want to just replace the `mf::LogInfo` with `mf::Verbatim` in the loop, or in both the look and the intro line, depending on the effect you want. Each of the [messagefacility streams](https://github.com/art-framework-suite/messagefacility/blob/23731098651df6a6087a2d4d697b4baadba5590d/messagefacility/MessageLogger/MessageLogger.h#L207-L219) comes with a version with the "decoration" and a version without it; `mf::LogVerbatim` is the twin of `mf::LogInfo`.

Reply:
Yes, that's precisely what I was trying to achieve to easily read the printed table for better debugging. I'll follow your suggestion and use one of those.